templates/_icons.css

```diff
@@ -1,11 +1,11 @@
 @font-face {
 	font-family: "<%= fontName %>";
-	src: url('<%= fontPath %><%= fontName %>.eot');
-	src: url('<%= fontPath %><%= fontName %>.eot?#iefix') format('eot'),
-		url('<%= fontPath %><%= fontName %>.woff2') format('woff2'),
-		url('<%= fontPath %><%= fontName %>.woff') format('woff'),
-		url('<%= fontPath %><%= fontName %>.ttf') format('truetype'),
-		url('<%= fontPath %><%= fontName %>.svg#<%= fontName %>') format('svg');
+	src: url('<%= fontPath %><%= fontName %>.eot<%= cacheBusterQueryString %>');
+	src: url('<%= fontPath %><%= fontName %>.eot?<%= cacheBuster %>#iefix') format('eot'),
```

Comment:
Why are you using assigned `var` `cacheBuster` instead of also using `cacheBusterQueryString ` ?